src/sentry/integrations/issues.py

```diff
@@ -228,6 +229,14 @@ def update_group_status(self, groups, status, activity_type):
     def sync_status_inbound(self, issue_key, data):
         if not self.should_sync('inbound_status'):
             return
+
+        organization = Organization.objects.get(id=self.organization_id)
+        has_issue_sync = features.has('organizations:integration:issue_sync',
+                                      organization)
```

Comment:
How does passing `actor=` to this in the endpoints, change the result (if at all) compared to not supplying an actor here?

Reply:
passing actor where possible is better because it means it will work for feature handlers that rely on actor (see https://github.com/getsentry/getsentry/blob/master/getsentry/features.py#L296 for an example)

however, this will primarily be used by evan's feature handler, which is based on org: https://github.com/getsentry/getsentry/pull/2147 so it doesn't really matter a whole lot